packages/ui/components/ScrollToTopUi/index.tsx

```
@@ -0,0 +1,34 @@
+const SCROLL_THRESHOLD = 150;
+const handleScroll = (event: React.UIEvent<HTMLDivElement>) => {
+      const scrollTop = event.currentTarget.scrollTop;
+       if (scrollTop > SCROLL_THRESHOLD) {
+       setOpenTopButton(true);
+        } else {
+        setOpenTopButton(false);
+        }
+     };
+return  <div className="space-y-6 max-h-[500px]  overflow-y-scroll no-scrollbar"  ref={endRef} onScroll={handleScroll}>
+            <div ref={startRef}></div>
+            <>{children}</>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)